torch/onnx/symbolic_helper.py

```diff
@@ -228,15 +228,12 @@ def _sort_helper(g, input, dim, decending=True, out=None):
 def _topk_helper(g, input, k, dim, largest=True, sorted=False, out=None):
     if out is not None:
         _unimplemented("TopK", "Out parameter is not supported")
+    if not _is_value(k):
+        k = g.op("Constant", value_t=torch.tensor(k, dtype=torch.int64))
+    k = g.op("Reshape", k, g.op("Constant", value_t=torch.tensor([1])))
```

Comment:
nit: `Reshape` can be put in an else branch. In the if branch construct the `Constant` with torch.tensor([k], dtype=torch.int64).

Reply:
That won't work for opset 10, if k is value. I updated this with a minor change.